app/src/main/java/io/homeassistant/companion/android/sensors/CarSensorManager.kt

```diff
@@ -108,13 +109,23 @@ class CarSensorManager :
         get() = R.string.sensor_name_car
 
     override suspend fun getAvailableSensors(context: Context): List<SensorManager.BasicSensor> {
-        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+        // TODO: show sensors for automotive (except odometer) once
+        //  we can ask for special automotive permissions in requiredPermissions
+        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O &&
+            !context.packageManager.hasSystemFeature(
+                    PackageManager.FEATURE_AUTOMOTIVE
+                )
+        ) {
             sensorsList
         } else {
             emptyList()
         }
     }
 
+    override fun hasSensor(context: Context): Boolean {
+        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
```

Comment:
We should add the automotive check here and I think we should also add a check to hide the sensors from the minimal version as well because those devices won't have the GMS permissions and thus wont be able to add any of the sensors.

Reply:
The sensor for the car model doesn't require any special permissions, but I guess we can still limit this to full version since android auto probably doesn't work on minimal

Reply:
> The sensor for the car model doesn't require any special permissions,

my bad forgot about that one 😅 

> but I guess we can still limit this to full version since android auto probably doesn't work on minimal

I _think_ its safe to assume it wont work on the minimal build until someone tells us it would?